test/salt/pillar/default.sls

```diff
@@ -67,5 +67,7 @@ proftpd:
   # Specify the UNIX Local Socket
   ClamLocalSocket: /var/run/clamav/clamd.sock
   Modules:
-    mod_radius: false
-    mod_ident: true
+    enabled:
+      - mod_ident
+    disabled:
+      - mod_radius
```

Comment:
As mentioned already, an end-user should not be forced to provide this block in their pillar/config, in order to use the values in `defaults.yaml`.